src/test_utils.rs

```diff
@@ -250,6 +250,17 @@ impl TestContext {
         Self::builder().configure_fiona().build().await
     }
 
+    /// Print current chat state.
+    pub async fn print_chats(&self) {
```

Comment:
The nested block_on calls are not possible in tokio anymore (it has a check to avoid them and panics). As this seems to be for debugging, I figured I add this method here, and allow devs to manually add it if they want this information when debugging.